versioned_docs/version-1.2/plugins/extra-plugins.md

```diff
@@ -814,6 +814,171 @@ table.insert(lvim.builtin.cmp.sources, 1, { name = "copilot" })
 	{"oberblastmeister/neuron.nvim"},
 ```
 
+### [Telekasten](https://github.com/renerocksai/telekasten.nvim)
+
+**A Neovim (lua) plugin for working with a markdown zettelkasten / wiki and mixing it with a journal, based on telescope.nvim**
+
+```lua
+	  { 'renerocksai/telekasten.nvim' },
+```
+
+```lua
+lvim.builtin.which_key.mappings.z = {
+  name = "+Telekasten",
+  t = { "<cmd>lua require('telekasten').toggle_todo()<CR>", "Toggle todo" },
+  f = { "<cmd>lua require('telekasten').find_notes()<CR>", "Find notes" },
+  d = { "<cmd>lua require('telekasten').find_daily_notes()<CR>", "Find daily notes" },
+  g = { "<cmd>lua require('telekasten').search_notes()<CR>", "Search notes" },
+  z = { "<cmd>lua require('telekasten').follow_link()<CR>", "Follow link" },
+  T = { "<cmd>lua require('telekasten').goto_today()<CR>", "Goto today" },
+  W = { "<cmd>lua require('telekasten').goto_thisweek()<CR>", "Goto thisweek" },
+  w = { "<cmd>lua require('telekasten').find_weekly_notes()<CR>", "Find weekly notes" },
+  n = { "<cmd>lua require('telekasten').new_note()<CR>", "New note" },
+  N = { "<cmd>lua require('telekasten').new_templated_note()<CR>", "New templated note" },
+  y = { "<cmd>lua require('telekasten').yank_notelink()<CR>", "Yank notelink" },
+  c = { "<cmd>lua require('telekasten').show_calendar()<CR>", "Show calendar" },
+  C = { "<cmd>CalendarT<CR>", "CalendarT" },
+  i = { "<cmd>lua require('telekasten').paste_img_and_link()<CR>", "Paste img and link" },
+  b = { "<cmd>lua require('telekasten').show_backlinks()<CR>", "Show backlinks" },
+  F = { "<cmd>lua require('telekasten').find_friends()<CR>", "Find friends" },
+  I = { "<cmd>lua require('telekasten').insert_img_link({ i=true })<CR>", "Insert img link" },
+  p = { "<cmd>lua require('telekasten').preview_img()<CR>", "Preview img" },
+  m = { "<cmd>lua require('telekasten').browse_media()<CR>", "Browse media" },
+  ['#'] = { "<cmd>lua require('telekasten').show_tags()<CR>", "Show tags" },
+  r = { "<cmd>lua require('telekasten').rename_note()<CR>", "Rename note" },
+  ["."] = { "<cmd>lua require('telekasten').panel()<CR>", "Show Panel" }
+}
+
+vim.keymap.set("i", ";[]", "<cmd>:lua require('telekasten').insert_link({ i=true })<CR>")
+vim.keymap.set("i", ";zt", "<cmd>:lua require('telekasten').toggle_todo({ i=false })<CR>")
+vim.keymap.set("i", ";#", "<cmd>lua require('telekasten').show_tags({i = true})<cr>")
+
+local home = vim.fn.expand("~/zettelkasten")
+require('telekasten').setup({
```

Comment:
Do you actually need the configuration inside `require('telekasten').setup` ?  How much of that setup is customized vs just a copy of the default setup?